pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -356,8 +353,7 @@ func (dswp *desiredStateOfWorldPopulator) checkVolumeFSResize(
 	pvc *v1.PersistentVolumeClaim,
 	volumeSpec *volume.Spec,
 	uniquePodName volumetypes.UniquePodName,
-	mountedVolumesForPod map[volumetypes.UniquePodName]map[string]cache.MountedVolume,
-	processedVolumesForFSResize sets.String) {
+	mountedVolumesForPod map[volumetypes.UniquePodName]map[string]cache.MountedVolume) {
```

Comment:
update the comment: marks this volume as fsResizeRequired in ASW. (this is no longer true?)